VirtoCommerce.Storefront/Domain/Cart/CartBuilder.cs

```diff
@@ -73,7 +77,7 @@ public virtual async Task LoadOrCreateNewTransientCartAsync(string cartName, Sto
         {
             var cacheKey = CacheKey.With(GetType(), "LoadOrCreateNewTransientCart", store.Id, cartName, user.Id, currency.Code, type);
             var needReevaluate = false;
-            Cart = await _memoryCache.GetOrCreateExclusiveAsync(cacheKey, async (cacheEntry) =>
+            Cart = await MemoryCacheExtensions.GetOrCreateExclusiveAsync(_memoryCache, cacheKey, async (cacheEntry) =>
```

Comment:
Thanks to that fact what the `IStorefrontMemoryCache` derived from `IMemoryChache` the `GetOrCreateExclusiveAsync` extension must also work for it. So you don't need to change this line at all.